kafka-ui-react-app/src/components/Brokers/BrokersList/BrokersList.tsx

```diff
@@ -98,10 +104,47 @@ const BrokersList: React.FC = () => {
               cell={cell}
               getValue={getValue}
               renderValue={renderValue}
+              renderSegments
             />
           ),
       },
-      { header: 'Segment Count', accessorKey: 'count' },
+      {
+        // eslint-disable-next-line react/no-unstable-nested-components
+        header: () => <SkewHeader />,
+        accessorKey: 'partitionsSkew',
+        // eslint-disable-next-line react/no-unstable-nested-components
+        cell: ({ getValue }) => (
+          <ColoredCell
+            value={getValue<number>() !== null ? `${getValue<number>()}%` : '-'}
+            warn={getValue<number>() >= 10 && getValue<number>() < 20}
+            attention={getValue<number>() >= 20}
+          />
+        ),
+      },
+      { header: 'Leaders', accessorKey: 'partitionsLeader' },
+      {
+        header: 'Leader skew',
+        accessorKey: 'leadersSkew',
+        // eslint-disable-next-line react/no-unstable-nested-components
+        cell: ({ getValue }) => (
+          <ColoredCell
```

Comment:
lets not call `getValue` multiple times lets just call it once and put it in a variable

Reply:
Updated